### ISSUE TITLE
[FR] new reset level: cookies

### ISSUE THREAD

**Issue by ildar** [tobykurien/WebApps#197]
It's disappointing that "Reset" doesn't reset cookies. Could you please make a checkbox for that? Thanks!

**Comment by ChristianTacke** [comment]
Reset could also have an option to clear the complete cache for that webapp?

**Comment by tobykurien** [comment]
You can delete, then re-add the WebApp

**Comment by ChristianTacke** [comment]
Well, only clearing the cache is different from clearing the cookies.
I recently had two times the issue, that things only worked, when I cleared the complete cache of the WebApps app (using system settings). So doing that more easily (on a per Webapp level) might be nicer.

**Comment by Andrewskiz** [comment]
I agree with this request, additionally I think an option to clear cookies on close would be nice.

**Comment by 01yakiek** [comment]
This would be utterly helpful. My current workaround is just to enter "/data/data/com.tobykurien.webapps\databases\webapps4.db", table "webapps", column "cookies", and replace it with empty string. Not so easily because I had to move the database folder out of system before I could save, and then move back to its original location.

Further more, it would be helpful to have a global unblocked domain management menu. In the same database, there is a table named "dmomain_names" which contains all unblocked domain linked to "webbappId".

Additionally, because not everything is sandboxed, some data gets carried over between same domains. Deleting "/data/com.tobykurien.webapps/app_webview" solves most problems, but in my case, deleting "/data/com.tobykurien.webapps/app_webview/Local Storage" is enough.

If the cookie is stored separately in databases unlike the "app_webview" directory mess, why the heck isn't a clear cookies a thing? Please don't suggest senseless workaround such as deleting the webapp because then we would have to login/unblock domains again.

Thanks.
